tests/test_cli.py

```diff
@@ -441,11 +446,50 @@ def test_cli_valid_scenarios(status, unit, exit_code, tmp_path):
             "validate-scenarios",
             str(tmp_path / f"{status}_data.xlsx"),
             "--definitions",
-            str(
-                MODULE_TEST_DATA_DIR
-                / "structure_validation_no_mappings"
-                / "definitions"
-            ),
+            str(MODULE_TEST_DATA_DIR / "structure_validation" / "definitions"),
+            "--dimension",
+            dimensions,
+        ],
+    )
```

Comment:
You'll have to expand the list by adding something like
```python
dimension_args = []
for dim in dimensions:
    dimension_args.append("--dimension")
    dimension_args.append(dim)
```
Maybe this can be done more elegantly...?

```suggestion
       ] + dimension_args
    )
```